src/dune

```diff
@@ -9,6 +9,8 @@
   decompress.lzo
   mirage-crypto
   tls
+  hxd.core
+  hxd.string
```

Comment:
do we really want this dependency in the core library?

Reply:
Currently, it's used by `pp_hum` of `Tls_crypt_v2.{Client,Server}`. It's not mandatory but the output is nicer than `ocaml-hex` I think. We can also not provide `pp_hum` and think about an other way to print out keys.

Reply:
I moved the `Tls_crypt_v2.Key.pp_hum` function into the application so the library can drop the hxd dependency. It is not used anywhere though.